src/plugins/ColorVisionPlugin.spec.js

```diff
@@ -53,4 +53,21 @@ describe('ColorVisionPlugin', () => {
 
     expect(cvp.colorDropdown.value).to.equal('tritanopia');
   });
+
+  it('Plugin should work without any controls', (done) => {
+    //set up empty plugin
+    cvp = new ColorVisionPlugin();
+    cvp.preload({ client: new Bellhop() });
+
+    // const emptyIframe = document.createElement('iframe');
```

Comment:
Should we maybe remove these commented lines if the code isn't going to be used?